plasmapy/dispersion/__init__.py

```diff
@@ -24,17 +24,19 @@ class DispersionOutput:
         Angular frequencies. Can be complex, and must be the same size as *k*.
     """
     def __init__(self, input, omega, k):
-        if omega.size != k.size:
+        if len(omega.shape) != 1:
+            raise ValueError('"omega" must be a 1D array (got {}D)'.format(len(omega.shape)))
+        if omega.shape[0] != k.shape[0]:
             raise ValueError(
-                'Arguments "omega" (size {}) and "k" (size {}) must have the same size.'.format(omega.size, k.size))
+                'The first dimension of "omega" and "k" must match (got {} and {} respectively)'.format(omega.shape[0], k.shape[0]))
```

Comment:
I'm noticing a very distinct and disturbing lack of f-strings here. :stuck_out_tongue_winking_eye:

Reply:
Do you have a link to a good f-string primer? Happy to use them but have never used them before!

Reply:
Oh man, you're gonna love those! Here's something I recall that popped up recently:

https://medium.com/@NirantK/best-of-python3-6-f-strings-41f9154983e

Reply:
Returning to this and glad to say I've now learnt f-strings 🎉